### DISCUSSION TITLE
Multi-rename needs more sophisticated skip/overwrite options

### DISCUSSION THREAD

**Discussion by audiomuze** [doublecmd/doublecmd#1070] [Suggestions]
### Description

Currenly when using multi-rename capability Double commander's options force you to manually process each overwrite/skip operation with no ability to apply some rules to automate skip/overwrite based on selected criteria:
![image](https://user-images.githubusercontent.com/6852511/210536405-7942880b-340a-43c9-9e25-0deff5152bcf.png)

Midnight Commander's skip/ overwrite options on the other hand allow for far more flexibility & automation based on selected criteria:
![image](https://user-images.githubusercontent.com/6852511/210536462-aab8877c-e0b4-44c6-b749-fbf16786d02a.png)

### Proposed technical implementation details (optional)

Implement the same overwrite options as are currently available to regular copy/move operations:

![image](https://user-images.githubusercontent.com/6852511/210537400-158c73c7-76f3-480b-8eb0-f76d6e4dcfc2.png)

**Comment by ags1234** [comment]
Midnight Commander has some useful and interesting options, especially "Don't overwrite with zero length file"

**Comment by audiomuze** [comment]
DC's options under Other are pretty handy also, they're just not universally exposed where copy/move operations are enabled.

**Comment by alexx2000** [comment]
I added several additional options.

**Reply by audiomuze** [reply]
Where are they, DC is still behaving exactly as I first described it?
